crates/biome_html_analyze/tests/specs/a11y/noSvgWithoutTitle/html/valid.html

```
@@ -0,0 +1,40 @@
+</svg>
+<svg role="img" aria-label="title">
+	<title id="title">Pass</title>
+</svg>
+<svg role="img" aria-label="title">
+	<span>Pass</span>
+</svg>
+<svg role="img" aria-label="title">
+	<span id="sample">Pass</span>
+</svg>
+<svg role="img" aria-labelledby="title">
+	<title id="title">Pass</title>
```

HTMLHint (1.9.2)

[error] The id value [ title ] must be unique.

(id-unique)